web_src/js/features/tableSort.js

```diff
@@ -0,0 +1,72 @@
+export default function initTableSort() {
+  $('th').each(function () {
+    // if data attribute sortt is set
+    if ($(this)[0].dataset.sortt) {
+      // get data
+      let data = $(this)[0].dataset.sortt;
```

Comment:
```suggestion
      let data = $(this)[0].dataset.sortt || '';
```

just in case this is falsy